api/v1alpha3/machineset_types.go

```diff
@@ -132,7 +132,7 @@ type MachineSetStatus struct {
 
 	// ObservedGeneration reflects the generation of the most recently observed MachineSet.
 	// +optional
```

Comment:
Should this be no longer optional, since we are not setting omitempty? https://github.com/eBay/Kubernetes/blob/master/docs/devel/api-conventions.md#optional-vs-required